eea/facetednavigation/widgets/widget.py

```diff
@@ -110,14 +110,21 @@ def css_class(self):
         """
         css_type = self.widget_type
         css_title = normalizer.normalize(self.data.title)
-        return 'faceted-{0}-widget section-{1}'.format(css_type, css_title)
+        custom_css = ' {}'.format(self.data.custom_css) if len(self.data.custom_css) else ''
+        return 'faceted-{0}-widget section-{1}{2}'.format(css_type, css_title, custom_css)
 
     @property
     def hidden(self):
         """ Widget hidden?
         """
         return self.data.hidden
 
+    @property
+    def custom_css(self):
+        """ Widget custom_css
+        """
+        return self.data.custom_css
```

Comment:
And also as property on Widget...

Reply:
We could eventually remove this property because we don't need to use it in the templates?